sapi/fpm/fpm/fpm_sockets.c

```diff
@@ -102,14 +98,15 @@ static void fpm_sockets_cleanup(int which, void *arg) /* {{{ */
 		free(ls->key);
 	}
 
-	if (env_value) {
+	if (env_str.c) {
+		smart_string_0(&env_str);
```

Comment:
Would:
```suggestion
		smart_string_appendc(&env_str, '\0');
```

for consistency.